C/eval.c

```diff
@@ -806,7 +806,7 @@ simplicity_err evalTCOExpression( flags_type anti_dos_checks, UWORD* output, con
   static_assert(2*DAG_LEN_MAX <= SIZE_MAX / sizeof(frameItem), "frames array does not fit in size_t.");
```

Comment:
There is a `calloc` in the line above, which is a problem, because we definitely shouldn't be mixing `simplicity_free(cell)` with `calloc(cells)`

Reply:
Seems there is a second `calloc` in this file as well.

Reply:
I replaced all occurrences of `calloc`. I included the occurrences in test.c for consistency.